packages/sra-report/src/types.ts

```diff
@@ -0,0 +1,3 @@
+export interface Schema {
```

Comment:
realized theres a `Schema` interface exported by our json schemas package, so I'll just use that